dialogTemplate/de.json

```diff
@@ -10,9 +10,10 @@
 				{
 					"value": "short",
 					"synonyms": [
+						"einfach",
 						"schnell",
-						"schnellen",
-						"adhoc"
+						"brief",
```

Comment:
not sure something was translated here, you don't use "brief" ("Brief" would be a letter)